recipes/easyhttpcpp/all/conanfile.py

```diff
@@ -49,6 +50,7 @@ def layout(self):
 
     def requirements(self):
         self.requires("poco/1.12.4", transitive_headers=True, transitive_libs=True)
+        self.requires("openssl/[>=1.1 <4]")
```

Comment:
Added this because the library is making _direct calls_ to openssl code:
https://github.com/sony/easyhttpcpp/blob/ef7dca78f76877d51b329464c322e87eef54195f/src/unix/SslContextCreatorImpl.cpp#L39-L40

Otherwise can fail with
```
[100%] Linking CXX shared library lib/libeasyhttp.dylib
Undefined symbols for architecture arm64:
  "_SSL_CTX_load_verify_locations", referenced from:
      easyhttpcpp::SslContextCreatorImpl::createContext(Poco::AutoPtr<easyhttpcpp::EasyHttpContext>) in SslContextCreatorImpl.cpp.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (us
```